### PR TITLE
Read & Save Event

### PR DESCRIPTION
**Pull request by diegomarvid**
I will create the following functionalities:

- [x] Read Event from CSV (pandas read csv wrapper).
- [x] Save Event to CSV (pandas save df to csv wrapper).
- [x] Add tests for both functionalities.

### CODE REVIEW

File: temporian/io/read_event.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    Raises:
+        FileNotFoundError: If file does not exist.
+        ValueError: If file is not a csv file.
+
+    """
+    # check file exists and is a csv
+    path = Path(path)
+    if not path.exists():
+        raise FileNotFoundError(f"File {path} does not exist.")
```
Comment (ianspektor) [inline temporian/io/read_event.py:33]: pandas already does this check, we can skip it

File: temporian/io/read_event.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    if path.suffix != ".csv":
+        raise ValueError(f"File {path} is not a csv file.")
```
Comment (ianspektor) [inline temporian/io/read_event.py:35]: pandas doesn't do this check, but I don't think we should either - the user might have valid csv content in a file marked with a .txt extension. I would just call pandas' method directly with whatever we receive as `path`.

File: temporian/io/read_event.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
+from typing import List, Optional
+import pandas as pd
+from pathlib import Path
+from temporian.implementation.numpy.data.event import NumpyEvent
+
+
+def read_event(
+    path: str,
```
[the remark this answers is not in the document]
Comment (diegomarvid) [reply temporian/io/read_event.py:8]: Can we just leave it to pandas read csv to check the input path?

[the remark this answers is not in the document]
Comment (ianspektor) [reply temporian/io/read_event.py:8]: imo yep

**Review by ianspektor — APPROVED** [review]

**Pull request merged**